x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/ExplainLifecycleIT.java

```diff
@@ -257,6 +258,66 @@ public void testExplainOrder() throws Exception {
         );
     }
 
+    public void testStepInfoPreservedOnAutoRetry() throws Exception {
+        String policyName = "policy-" + randomAlphaOfLength(5).toLowerCase(Locale.ROOT);
+
+        Request createPolice = new Request("PUT", "_ilm/policy/" + policyName);
+        createPolice.setJsonEntity(
+            "{"
+                + "\"policy\": {"
+                + "  \"phases\": {"
+                + "    \"hot\": {"
+                + "      \"actions\": {"
+                + "        \"rollover\": {"
+                + "          \"max_docs\": 1"
+                + "        }"
+                + "      }"
+                + "    }"
+                + "  }"
+                + "}"
+                + "}"
+        );
```

Comment:
Nit: I find `"""` a bit  more readable in case you want to give it a try.

Reply:
Ohh yeah we're targeting > Java 15 aren't we :D 100% I'll switch it over for a multiline string

Reply:
Done